bin/btc-server/src/util.rs

```diff
@@ -11,31 +15,39 @@ use bitcoin::{
 };
 use frost_secp256k1_tr as frost;
 use futures_util::Future;
-use lazy_static::lazy_static;
 use log::{error, info};
 use std::{
     collections::{HashMap, HashSet},
-    time::Duration,
-};
-
-use crate::{
-    database::{self, Db, Utxo},
-    pegout_id::PegoutId,
+    time::{Duration, SystemTime, UNIX_EPOCH},
 };
 use thiserror::Error;
 
-lazy_static! {
-    static ref MAX_FEERATE: bitcoin::FeeRate =
-        bitcoin::FeeRate::from_sat_per_vb(300).expect("valid feerate");
-}
-
 // Psbt validation flags
 pub(crate) const NO_FLAGS: u8 = 0u8;
 pub(crate) const ROUND1: u8 = 1u8;
 pub(crate) const ROUND1_TRANSITION: u8 = (1u8 << 1) | ROUND1;
 pub(crate) const ROUND2: u8 = (1u8 << 2) | ROUND1_TRANSITION;
 pub(crate) const ROUND2_TRANSITION: u8 = (1u8 << 3) | ROUND1_TRANSITION;
 
+/// Checks if the age of a block, based on its timestamp, is within an acceptable duration.
+/// # Arguments
+///
+/// * `timestamp` - The timestamp of the block in seconds since the UNIX epoch.
+/// * `max_age_cutoff` - The maximum acceptable age of the block as a `Duration`.
+///
+/// # Returns
+///
+/// Returns `true` if the block's age is less than the specified max cutoff age, otherwise `false`.
+pub fn is_block_age_acceptable(timestamp: u64, max_age_cutoff: Duration) -> bool {
+    let now = match SystemTime::now().duration_since(UNIX_EPOCH) {
+        Ok(now) => now.as_secs(),
+        Err(_) => return false,
+    };
+
+    let threshold = now.saturating_sub(max_age_cutoff.as_secs());
+    timestamp > threshold
```

Comment:
nit: the existing logic is correct, but since the method refers to 'age', i think this way round might be clearer
```suggestion
    let age_secs = now.saturating_sub(timestamp);
    age_secs < max_age_cutoff.as_secs()
```

Reply:
removed this method since it wasn't being used